python/tvm/relay/frontend/keras.py

```diff
@@ -1266,11 +1266,15 @@ def _convert_layer(keras_layer, etab, scope=""):
             # - In Keras, model._network_nodes contains keys of all nodes relevant to the
             #   current model;
             # - In tf.Keras, this is already done as part of tensorflow.keras.network.get_config
-            if (
-                not is_tf_keras
-                and not model._node_key(keras_layer, node_idx) in model._network_nodes
-            ):
-                continue
+            if not is_tf_keras:
+                if (
```

Comment:
Can you put a comment on which branch refers to which version, e.g `# TFlite 2.6` ?

Reply:
Done